composer.json

```diff
@@ -19,7 +19,10 @@
     "minimum-stability": "dev",
     "prefer-stable": true,
     "require": {
-        "php": "^7.4|^8.0"
+        "php": "^7.4|^8.0",
+        "guzzlehttp/guzzle": "^7.3",
```

Comment:
I'm not sure it is a good idea to enforce usage of Guzzle. How about using PSR-18 instead?

Reply:
Agreed, but
```
$ composer req sentry/sentry
Using version ^3.3 for sentry/sentry
./composer.json has been updated
Running composer update sentry/sentry
Loading composer repositories with package information
Updating dependencies
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - sentry/sentry[dev-master, 3.3.0] require php-http/async-client-implementation ^1.0 -> could not be found in any version, but the following packages provide it:
      - symfony/http-client Provides powerful methods to fetch HTTP resources synchronously or asynchronously
      - php-http/guzzle6-adapter Guzzle 6 HTTP Adapter
      - symfony/symfony The Symfony PHP framework
      - php-http/curl-client PSR-18 and HTTPlug Async client with cURL
      - php-http/guzzle7-adapter Guzzle 7 HTTP Adapter
      - php-http/mock-client Mock HTTP client
      - voku/httpful A Readable, Chainable, REST friendly, PHP HTTP Client
      - php-http/react-adapter React HTTP Adapter
      - swisnl/php-http-fixture-client Fixture client for PHP-HTTP
      - phpgt/fetch Asynchronous HTTP client with promises for PHP 7 applications.
      - betalabs/guzzle6-adapter Guzzle 6 HTTP Adapter
      - neur0toxine/pock PSR-18 compatible HTTP mock library
      - nessworthy/amphp-http-httplug A small crappy adapter package to support php-http/async-client-implementation via. amphp/http-clien
      - phpgt/curl cURL object wrapper.
      - john_zuk/php-nbp-api NBP (Narodowy Bank Polski) API client
      - christophwurst/nextcloud-http-client HTTPlug and PSR7 adapter for Nextcloud
      - devster/http-client Symfony HttpClient component
      - gogoprint/guzzle6-adapter Guzzle 6 HTTP Adapter
      - pbxapi/php-http-fixture-client Fixture client for PHP-HTTP
      - phpgt/curlinterface cURL object wrapper.
      ... and 2 more.
      Consider requiring one of these to satisfy the php-http/async-client-implementation requirement.
    - sentry/sentry 3.3.x-dev is an alias of sentry/sentry dev-master and thus requires it to be installed too.
    - Root composer.json requires sentry/sentry ^3.3 -> satisfiable by sentry/sentry[3.3.0, 3.3.x-dev (alias of dev-master)].

Potential causes:
 - A typo in the package name
 - The package is not available in a stable-enough version according to your minimum-stability setting
   see <https://getcomposer.org/doc/04-schema.md#minimum-stability> for more details.
 - It's a private package and you forgot to add a custom repository to find it

Read <https://getcomposer.org/doc/articles/troubleshooting.md> for further common problems.

Installation failed, reverting ./composer.json and ./composer.lock to their original content.
```

Reply:
Well, you can add Guzzle as a dev dependency, but not as production one.

Reply:
Done